requirements/constraints.txt

```diff
@@ -87,3 +87,7 @@ numpy<1.17.0
 #  jsondiff==1.1.1 (from moto==1.2.0->-r requirements/edx/testing.in (line 33))
 #  jsondiff==1.2.0 (from -r requirements/edx/base.txt (line 146))
 jsondiff==1.1.1
+
+# 5.0.0.124 is raising 500 errors
+# TODO: remove this once the problem is fixed
+newrelic<5.0
```

Comment:
@waheedahmed is there a ticket created with the details of what went wrong?

Reply:
Not yet but I can create one, under which project? PROD?

Reply:
Yes, please create a ticket under prod for fixing this.

Reply:
wil do, thanks!